app/router.cjsx

```diff
@@ -232,6 +234,8 @@ module.exports =
       <Route path="project_status" component={ProjectStatusList} />
       <Route path="project_status/:owner/:name" component={ProjectStatus} />
       <Route path="grantbot" component={Grantbot} />
+      <Route path="organization_status" component={OrganizationStatusList} />
+      <Route path="organization_status/:owner/:name" component={OrganizationStatus} />
```

Comment:
Same comment about the URLs here. Dashes should be preferred. URLs for projects should fixed too, but that's out of scope for this PR.